README.md

```diff
@@ -48,70 +48,80 @@ Reward formula: `((count * wordValue) * (score * formattingMultiplier) * n) * re
 
 Here is a possible valid configuration to enable this plugin. See [these files](./src/configuration) for more details.
 
-
 ```yaml
 plugin: ubiquibot/conversation-rewards
 with:
-    evmNetworkId: 100
-    evmPrivateEncrypted: "encrypted-key"
-    erc20RewardToken: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
-    dataCollection:
-      maxAttempts: 10
-      delayMs: 10000
-    incentives:
-      requirePriceLabel: true
-      contentEvaluator:
-      userExtractor:
-        redeemTask: true
-      dataPurge:
-      formattingEvaluator:
-        scores:
-          br: 0
-          code: 1
-          p: 1
-          em: 0
-          img: 0
-          strong: 0
-          blockquote: 0
-          h1: 1
-          h2: 1
-          h3: 1
-          h4: 1
-          h5: 1
-          h6: 1
-          a: 1
-          li: 1
-          td: 1
-          hr: 0
+  evmNetworkId: 100
+  evmPrivateEncrypted: "encrypted-key"
+  erc20RewardToken: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
+  dataCollection:
+    maxAttempts: 10
+    delayMs: 10000
+  incentives:
+    requirePriceLabel: true
+    contentEvaluator:
+      multipliers:
+        - select: [ISSUE_SPECIFICATION]
+          relevance: 1
+        - select: [PULL_AUTHOR]
+          relevance: 1
+        - select: [PULL_ASSIGNEE]
+          relevance: 1
+        - select: [PULL_COLLABORATOR]
+          relevance: 1
+        - select: [PULL_CONTRIBUTOR]
+          relevance: 1
+    userExtractor:
+      redeemTask: true
```

Comment:
What is this? We should just make it a single property without nesting probably.

Reply:
The nesting is die to `userExtractor` being its own module and made it clearer that this module is the only one using these variables.
`redeemTask` means the tasks are redeemable e.g. the reward can be collected.